Update README.md (#11)
cloudflare/worker-template-router: README.md

[Demo /bar](http://workers-tooling.cf/demos/router/bar) | [Demo /foo](http://workers-tooling.cf/demos/router/foo)

#### Wrangler

You can use [wrangler](https://github.com/cloudflare/wrangler) to generate a new Cloudflare Workers project based on this template by running the following command from your terminal:

```
wrangler generate myApp https://github.com/cloudflare/worker-template-router
```

> Before publishing your code you need to edit `wrangler.toml` file and add your `account_id` (Your Cloudflare account ID) number, for more information about publishing your code [check](https://workers.cloudflare.com/docs/quickstart/configuring-and-publishing/) documentaion.

Reviewer: ```suggestion
Before publishing your code you need to edit `wrangler.toml` file and add your Cloudflare `account_id` - more information about publishing your code can be found [in the documentation](https://workers.cloudflare.com/docs/quickstart/configuring-and-publishing/).
```

Author: Sounds even better than mine 👍
